cluster/install.go

```diff
@@ -1,26 +0,0 @@
-package cluster
-
-import (
-	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/apimachinery/pkg/runtime/schema"
-
-	clusterv1 "github.com/open-cluster-management/api/cluster/v1"
-)
-
-const (
-	GroupName = "cluster.open-cluster-management.io"
-)
-
-var (
```

Comment:
Was this meant to be deleted?

Reply:
yes, I think it is not necessary? The Install is cluster/v1/register.go. I do not think we need another func here.